ci-operator/step-registry/servicemesh/kiali/ossm-cypress-tests/execute/servicemesh-kiali-ossm-cypress-tests-execute-commands.sh

```
@@ -1,89 +1,60 @@
+oc patch kiali kiali -n "${ISTIO_NAMESPACE}" -p "{\"spec\":{\"external_services\":{\"grafana\": {\"enabled\": true}}}}" --type=merge
 
 sleep 120
-oc wait --for condition=Successful kiali/kiali -n ${ISTIO_NAMESPACE} --timeout=250s
-oc wait --for condition=available deployment/kiali -n ${ISTIO_NAMESPACE} --timeout=250s
+oc wait --for condition=Successful kiali/kiali -n "${ISTIO_NAMESPACE}" --timeout=250s
+oc wait --for condition=available deployment/kiali -n "${ISTIO_NAMESPACE}" --timeout=250s
 
-export CYPRESS_BASE_URL="${CONSOLE_URL}"
-export CYPRESS_USERNAME=${OCP_CRED_USR}
-export CYPRESS_PASSWD=${OCP_CRED_PSW}
+export CYPRESS_BASE_URL=$(oc whoami --show-console)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)